pandas/core/series.py

```diff
@@ -840,11 +840,15 @@ def _slice(self, slobj: slice, axis: int = 0) -> "Series":
 
     def __getitem__(self, key):
         key = com.apply_if_callable(key, self)
+        key = lib.item_from_zerodim(key)
 
         if key is Ellipsis:
             return self
 
-        key_is_scalar = is_scalar(key)
+        # check for is_list_like/slice instead of is_scalar to allow non-standard
+        #  scalars through, e.g. cftime.datetime needed by xarray
+        #  https://github.com/pydata/xarray/issues/3751
+        key_is_scalar = not is_list_like(key) and not isinstance(key, slice)
```

Comment:
This sneaked in from a different issue / pr?

Reply:
looks like it, will revert, thanks